Enhance Profile Image Upload Process with formatting option #137

ShristiSharan opened the pull request:
Enhanced the Image Uploading Process & made it smooth out

1. Direct Navigating to local storage and uploading procedure
2. Cropper added to adjust the frame wisely
3. Quick reflection of the updated upload image.

Closes:#137

Reference of my change

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/fbdf37ee-73e7-49d1-9fc7-3f70db3c271a)

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/4ebcead6-cf94-440c-ad2b-988db6fafc8a)

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/f7f36d0d-bc4d-43e5-95d3-c7a6d809f98f)

and the change is quickly updated- 
![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/6118e0e2-ef9a-4c12-9cca-1e121674e5b0)

ShristiSharan commented:
Kindly review my PR @SamarthKadam

ShristiSharan commented:
Kindly @SamarthKadam Merge this PR, enhanced

SamarthKadam commented:
@ShristiSharan make the edit icon little smaller and also make sure those buttons like upload and cancel make them synchronized with the rest of the buttons like update and reset

ShristiSharan commented:
> @ShristiSharan make the edit icon little smaller and also make sure those buttons like upload and cancel make them synchronized with the rest of the buttons like update and reset. And also the crop button looks like a square. Make it look like a circle as our dp appears to be in circle

Sure I'll fix it!

ShristiSharan commented:
Fixed

- [x] Pen icon smaller
- [x] Modified Update and Cancel button synchronised with other buttons.
- [x] Cropper on Preview Image now appears to be circular highlighting how dp will finally look alike.

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/0f156001-85e3-4c5c-9c0c-a9b0dd49080c)

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/1fb78ea2-785f-40d5-97cc-534c459bf5cb)

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/acb5b162-5ed3-4be1-a02e-0f5950985e3a)

@SamarthKadam kindly merge this PR

SamarthKadam commented:
@ShristiSharan why does it blur the background. Keep only the image editable not the whole screen

ShristiSharan commented:
> @ShristiSharan why does it blur the background. Keep only the image editable not the whole screen

Ya!....changed it now, I mistook that.

![image](https://github.com/SamarthKadam/ChatBox/assets/95679078/5d7c42e6-7028-4b98-8a07-6ba310a38936)

ShristiSharan commented:
@SamarthKadam , kindly check it as the changes are done.

ShristiSharan commented:
the branch is updated!....kindly merge this PR @SamarthKadam

ShristiSharan commented:
@SamarthKadam could you please merge this PR! otherwise i'll have to keep updating my branch upon incoming pulls

SamarthKadam commented:
@ShristiSharan Deployments have been failed have a look

ShristiSharan commented:
Done!!.... due to merge conflicts and @babel plugins it got erased.

ShristiSharan commented:
@SamarthKadam now to go... please check

ShristiSharan commented:
can you share the log? for deployment fail?
